src/tests/ftest/daos_test/daos_core_test.yaml

```diff
@@ -89,9 +89,6 @@ daos_tests:
     test_K:
       daos_test: K
       test_name: DAOS KV tests
-    test_F:
```

Comment:
i think it's fine to keep this test running in daos_core_test with a single client.